packages/admin-ui-docs/docs/guidelines/experimental/date-picker.mdx

```diff
@@ -45,9 +49,70 @@ function Example() {
 render(<Example />)
 ```
 
+### Helper Text
+
+```jsx noInline live
+const DatePickerField = experimental_DatePickerField
+const DatePickerCalendar = experimental_DatePickerCalendar
+const useDatePickerState = experimental_useDatePickerState
+
+function Example() {
+  const state = useDatePickerState()
+
+  return (
+    <>
+      <DatePickerField
+        label="Date"
+        state={state}
+        helperText="This is a helper text"
+      />
+      <DatePickerCalendar state={state} />
+    </>
+  )
+}
+
+render(<Example />)
+```
+
+### Min and Max values
+
+```jsx noInline live
+const DatePickerField = experimental_DatePickerField
+const DatePickerCalendar = experimental_DatePickerCalendar
+const useDatePickerState = experimental_useDatePickerState
+
+function Example() {
+  const state = useDatePickerState({
+    minValue: {
+      year: 2022,
+      month: 0,
+      day: 20,
+    },
+    maxValue: {
+      year: 2024,
+      month: 11,
+      day: 20,
+    },
+  })
+
+  return (
+    <>
+      <DatePickerField
+        label="Date"
+        state={state}
+        criticalText="Date out of bounds"
+      />
+      <DatePickerCalendar state={state} />
+    </>
+  )
+}
+
+render(<Example />)
+```
+
 ### Validation
 
-Use the `invalid` boolean to trigger the invalid state. It is up to you to define what is valid or not.
+Use the `invalid` boolean to trigger the invalid state. It is up to you to define what is valid or not. Additionally, you can add a `criticalText` to be displayed whether invalid.
```

Comment:
Our inputs use the `tone="critical"` to represent this state, what do you think of maintaining them in sync? I mean that we can change either here to this approach or the inputs to have an `invalid` property

Reply:
Changed to tone.